src/shared/js/components/Sidebar/Sidebar.js

```diff
@@ -115,7 +116,7 @@ function Sidebar({ defaultTree }) {
   };
 
   return (
-    <div className="csk-element-sidebar">
+    <div className={`csk-element-sidebar ${show && 'show'}`}>
```

Comment:
This will add a class 'false' to the element

Reply:
when show === false

Reply:
I'll update it.